File: HackerRank-10-Days-Statistics/Day 4/Day_4_Geometric_Distribution_II.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include<bits/stdc++.h>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    float num, den;
    int n;
    cin>>num>>den>>n;
    float p_d=num/den;
    float sum=0;
    
    for(int i=0;i<n;i++){
        sum+=pow(1-p_d,i)*p_d;
    }  
    cout<<fixed<<setprecision(3)<<sum<<endl;
    return 0;
}

File: HackerRank-10-Days-Statistics/Day 4/Day_4_Geometric_Distribution_II.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
import sys
a,b =sys.stdin.readlines()
num, den=list(map(int,a.strip().split())) 
n=int(b)
p_d=num/den

summation=0

for i in range(0,n):
    summation+=(1-p_d)**i * p_d
    
print(round(summation,3))
